### PR TITLE
Fix rotation.csv

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

No more long weeks and week-end work :stuck_out_tongue_closed_eyes: 

/cc @sbwsg @afrittoli 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: buildbot/rotation.csv [hunk 1]
```diff
@@ -10,359 +10,359 @@ Date,User
...
-2019-12-24,
-2019-12-25,vdemeest
-2019-12-26,andrea.frittoli
-2019-12-27,dibyo
-2019-12-28,christiewilson
-2019-12-29,sbws
+2019-12-21,
+2019-12-22,
+2019-12-23,vdemeest
+2019-12-24,andrea.frittoli
```
Comment (afrittoli) [inline buildbot/rotation.csv]: Uhm, not likely :P

Comment (vdemeester) [reply buildbot/rotation.csv]: lol :hugs: 
We can remove those or just not caring about them :angel:

**Pull request merged**